src/ci/resources/papi_v2alpha1_application.conf

```diff
@@ -1,194 +1,71 @@
-include required(classpath("application.conf"))
-include "build_application.inc.conf"
-include "papi_application.inc.conf"
-
-services {
-  MetadataService {
-    class = "cromwell.services.metadata.hybridcarbonite.HybridMetadataServiceActor"
-    config {
-      # This section may also contain the same set of options as would be present in the 'config' section of the
-      # default (cromwell.services.metadata.impl.MetadataServiceActor) config
-
-      # The carbonite section contains carbonite-specific options
-      carbonite-metadata-service {
-        # Enable carboniting
-        enabled = true
-
-        # Only carbonite workflows whose summary entry IDs are greater than or equal to this value:
-        minimum-summary-entry-id = 0
-
-        # To help us stay below the Travis "max log size" value:
-        debug-logging = false
-
-        # Which bucket to use for storing the generated metadata JSON
-        bucket = "carbonite-test-bucket"
-
-        # A filesytem able to access the specified bucket:
-        filesystems {
-          gcs {
-            # A reference to the auth to use for storing and retrieving metadata:
-            auth = "service_account"
-          }
-        }
-
-        # The current Centaur Carbonite logic times out at 30 seconds waiting for terminal workflows to be Carbonited.
-        # Set the backoff max interval for Carbonite freeze scanning below that limit.
-        freeze-scan {
-          initial-interval = 2 seconds,
-          max-interval = 15 seconds,
-          multiplier = 1.1
-        }
-      }
-    }
-  }
-
-  HealthMonitor.config.check-papi-backends: [
-    "Papi",
-    "Papiv2",
-    "Papiv2USADockerhub",
-    "Papiv2NoDockerHubConfig",
-    "Papiv2RequesterPays",
-    "Papi-Caching-No-Copy",
-    "Papiv2-Virtual-Private-Cloud",
-    "Papiv2-Retry-With-More-Memory",
-  ]
-}
+include required(classpath("papi_v2_shared_application.conf"))
```

Comment:
Looking at the failed tests I think this might should be 
`include required("papi_v2_shared_application.conf")` or `include "papi_v2_shared_application.conf"`